src/test/java/core/basesyntax/service/impl/CsvWriterServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvWriterServiceImplTest {
+    private static CsvWriterServiceImpl csvWriterService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        csvWriterService = new CsvWriterServiceImpl();
+    }
+
+    @Test
+    public void writeToFile_Ok_notOk() {
```

Comment:
так Ok или notOk? Давай переименуем метод
```suggestion
    public void writeToFile_emptyPath_notOk() {
```